modules/rules_erlang/3.9.0/MODULE.bazel

```diff
@@ -0,0 +1,72 @@
+module(
+    name = "rules_erlang",
+    compatibility_level = 3,
+    version = "3.9.0",
+)
+
+bazel_dep(
+    name = "rules_go",
+    repo_name = "io_bazel_rules_go",
+    version = "0.37.0",
+)
+bazel_dep(
+    name = "rules_pkg",
+    version = "0.5.1",
+)
+bazel_dep(
+    name = "bazel_skylib",
+    version = "1.3.0",
+)
+bazel_dep(
+    name = "gazelle",
+    repo_name = "bazel_gazelle",
+    version = "0.28.0",
+)
+
+# The union of go deps from bazel-gazelle and this repo
+go_deps = use_extension("@bazel_gazelle//:extensions.bzl", "go_deps")
+
+go_deps.from_file(go_mod = "@rules_erlang//:go.mod")
+
+use_repo(
+    go_deps,
+    "com_github_bazelbuild_buildtools",
+    "com_github_bmatcuk_doublestar_v4",
+    "com_github_fsnotify_fsnotify",
+    "com_github_emirpasic_gods",
+    "com_github_google_go_cmp",
+    "com_github_pelletier_go_toml",
+    "com_github_pmezard_go_difflib",
+    "in_gopkg_yaml_v2",
+    "org_golang_x_mod",
+    "org_golang_x_sync",
+    "org_golang_x_tools",
+    # Used internally by the go_deps module extension.
+    "bazel_gazelle_go_repository_directives",
```

Comment:
You should not need or use this repo, it is only consumed by the module extension itself.